cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -59,6 +61,8 @@ type GceManager struct {
 
 	service    *gce.Service
 	cacheMutex sync.Mutex
+	zone       string
```

Comment:
What about multi-zone clusters?

Reply:
They won't be fully supported in the first version. NAP won't create regional multiple node pools.